pagopa/io-app
# feat: [IOBP-137] Add barcode scan camera marker animation

## Short description
This PR adds the animation on the barcode camera scan marker

https://github.com/pagopa/io-app/assets/6160324/fac469c4-5150-4c8a-99bf-5771a81cd6c6

## List of changes proposed in this pull request
- added `useSineWaveAnimation` hook which handles the sine wave animation of the barcode line
- added `AnimatedCameraMarker` component

## How to test
To test this PR, open the barcode scan screen and check if the marker is animated. The animation should stop when the scanner is idle (5 seconds before a scan).
